css/components/pagination.postcss

```diff
@@ -0,0 +1,44 @@
+/*----------------------------------------*\
+  PAGINATION
+\*----------------------------------------*/
+
+/* TYPES */
+.pagination {
+    @apply inline-flex items-center;
+    @apply px-[1em];
```

Comment:
Is there a reason for not using px-4?

Reply:
I use the same code as the btn component but I think it's better to use px-4. I change it.